myhpom/templates/myhpom/dashboard.html

```diff
@@ -6,88 +6,111 @@
                     </div>
                     <div>
-                        <a href="/" class="small">
-                            Learn how you can be a donor.
-                        </a>
+                        <div class="dashboard-profile-information__organ-donor-status-info">
+                            <span class="h5 dashboard-profile-information__status">I am not an organ donor.</span>
+                        </div>
+                        <div>
+                            <a href="/" class="dashboard-profile-information__organ-donor-link">
+                                Learn how you can be a donor.
+                            </a>
+                        </div>
+                    </div>
+                    <div class="h6 dashboard-profile-information__organ-donor-edit-link">
+                        <a href="/" class="small"><span class="oi oi-pencil"></span> Edit</a>
                     </div>
                 </div>
             </div>
         </section>
 
         <hr/>
 
-        <section class="dashboard__section dashboard-health-network">
+        <section class="mb-5 dashboard__section dashboard-health-network">
             <div class="h6 float-right">
-                <a href="/" class="small">Update My Health Network</a>
+                <a href="/" class="small"><span class="oi oi-pencil"></span> Edit</a>
             </div>
 
             <h2 class="h6 dashboard__section-title">
                 Health network
             </h2>
 
-            <div class="bg-light dashboard-health-network__info-box">
-                <div class="h3">{% firstof request.user.userdetails.health_network.name request.user.userdetails.custom_provider '(Unknown Network)' %}</div>
-                <div class="small">
-                    Last Updated: January 1, 1970
+            <div class="dashboard__section-content">
+                <div class="dashboard-health-network__info-box">
+                    <div class="h3 dashboard-health-network__network-name">{% firstof request.user.userdetails.health_network.name request.user.userdetails.custom_provider '(Unknown Network)' %}</div>
+                    <div>
+                        Last Updated: January 1, 1970
+                    </div>
                 </div>
             </div>
         </section>
+
+        <hr/>
+
+        <sidebar class="row dashboard-faq">
+            <div class="col-sm-12">
+                <h2 class="h6 dashboard-faq__title">
+                    Frequently asked questions
+                </h2>
+            </div>
+            <div class="col-md-6 small">
+                <strong class="h6">{% lorem 5 w %}?</strong>
+                <p>
+                    {% lorem 24 w %}
+                </p>
+            </div>
+            <div class="col-md-6 small">
+                <strong class="h6">{% lorem 5 w %}?</strong>
+                <p>
+                    {% lorem 24 w %}
+                </p>
+            </div>
```

Comment:
I think the FAQ headings are $brand-blue instead of $brand-blue-medium, but this doesn't apply globally to .h6. So,

.dashboard-faq__heading {
    color: $brand-blue;
}